packages/core/src/Stores/client-store.js

```diff
@@ -1236,12 +1236,13 @@ export default class ClientStore extends BaseStore {
     }
 
     responseAuthorize(response) {
-        this.accounts[this.loginid].email = response.authorize.email;
-        this.accounts[this.loginid].currency = response.authorize.currency;
-        this.accounts[this.loginid].is_virtual = +response.authorize.is_virtual;
-        this.accounts[this.loginid].session_start = parseInt(moment().utc().valueOf() / 1000);
-        this.accounts[this.loginid].landing_company_shortcode = response.authorize.landing_company_name;
-        this.accounts[this.loginid].country = response.country;
+        const new_accounts = Object.create(this.accounts);
```

Comment:
@niloofar-deriv as checked we should not just create this object here and leave it and  dont reflect it on other sections.
in `this.account[this.loginid].something = response.something` `this.account` will of course used in some other sections.
so i dont think that this can work!
if we want to to use this approach we need to apply in the whole file. and not just in this section.

Reply:
I agree. beside that, I don't think this method helps to fix this issue, because when we are passing `this.account` to it, it will use `this.account` as the prototype for the new object so still an attacker can do prototype pollution if that's the reason we are getting security warning in the first place :thinking:

Reply:
I agree. beside that, I don't think this method helps to fix this issue, because as I understood, when we are passing `this.account` to `Object.create()`, it will use `this.account` as the prototype for the new object so still an attacker can do prototype pollution if that's the reason we are getting security warning in the first place :thinking:

using this method is only helpful when we pass null as the prototype for the new object so that it won't have access to other shared properties in the prototype.

![image](https://github.com/binary-com/deriv-app/assets/100833613/71a3e917-bf73-4331-ba7c-5dace6da9c17)

Reply:
not sure if I've understood the issue correctly or not. please correct me if I'm wrong

Reply:
Thank u guys for reviewing this,
As far as I checked this small change helped to fix the issue,
since we don't have this issue in the whole file we don't need to apply it everywhere , doing that may break something in our stores :smiling_face_with_tear: 
BTW if you have a better idea for fixing this let me know I'll apply them :pray:

Reply:
The previous fix was linked to this bug though: https://github.com/binary-com/deriv-app/commit/66acce03fe432a0664bc1fce40b03c32de1f6a99